### PR TITLE
Navigation Box made

### PR DESCRIPTION
**Pull request by Asypher00**
check /components/navigation.css for the code
Screenshot (13)

### CODE REVIEW

File: Navigation.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/228add9a8b.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/components/Navigation.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="nav-container">
```
Comment (RahulSingh9131) [inline Navigation.html]: use nav tag for keeping this code semantically correct.

File: Navigation.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <div class="nav-header">
+        <div class="nav-child"><i class="fas fa-bars"></i>
+        </div>
+        <div>Neeraj Agrawal</div></div>
+        <div class="nav-footer">
+        <div class="nav-child"><i class="fas fa-store fa-lg"></i></div>
+        <div class="nav-child"><i class="fas fa-shopping-cart fa-lg"></i></div>
+        <div class="nav-child"><i class="fas fa-heart fa-lg"></i></div>
```
Comment (RahulSingh9131) [inline Navigation.html]: if possible wrap these icons inside anchor tag. It will help you to add pages in it by using href.

**Review by RahulSingh9131 — COMMENTED** [review]
suggested few changes.

File: Navigation.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/228add9a8b.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/components/Navigation.css">
+    <title>Document</title>
```
Comment (Sameony) [inline Navigation.html]: Give appropriate Title to the document

File: components/Navigation.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
+body {
+    background-color : lightgray;
+
```
Comment (Sameony) [inline components/Navigation.css]: remove blank spaces and give proper alignment before merging
you can use prettier or beautify for better ease.

**Review by Sameony — APPROVED** [review]
suggested minor changes, however, do follow up on them
proper alignment and refactoring becomes a necessity especially in larger blocks of codes.

**Pull request merged**